x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/prebuilt_rules_preview.cy.ts

```diff
@@ -1049,5 +1062,104 @@ describe('Detection rules, Prebuilt Rules Installation and Update workflow', ()
         }
       );
     });
+
+    describe('Viewing rule changes in JSON diff view', { tags: TEST_ENV_TAGS }, () => {
+      it('User can see changes in a side-by-side JSON diff view', () => {
+        clickRuleUpdatesTab();
+
+        openRuleUpdatePreview(OUTDATED_RULE_1['security-rule'].name);
+        assertSelectedPreviewTab('JSON view');
+
+        cy.get(UPDATE_PREBUILT_RULE_PREVIEW)
+          .contains('Current rule')
+          .as('currentRuleColumnHeading');
+        cy.get(UPDATE_PREBUILT_RULE_PREVIEW)
+          .contains('Elastic update')
+          .as('elasticUpdateColumnHeading');
+
+        cy.get('@currentRuleColumnHeading').should('be.visible');
+        cy.get('@elasticUpdateColumnHeading').should('be.visible');
+
+        /* "Current rule" should be the left column, "Elastic update" should be the right column. */
+        cy.get('@currentRuleColumnHeading').then(($currentRuleColumnHeading) => {
+          const currentRuleLeftOffset = $currentRuleColumnHeading.offset()?.left || 0;
+          cy.get('@elasticUpdateColumnHeading').then(($elasticUpdateColumnHeading) => {
+            const elasticUpdateLeftOffset = $elasticUpdateColumnHeading.offset()?.left || 0;
+            expect(currentRuleLeftOffset).to.be.lessThan(elasticUpdateLeftOffset);
+          });
+        });
```

Comment:
I feel this positioning testing can be brittle, and not really needed. For the headers here, we are just testing an implementation detail of a EUI component. I think the tests above `cy.get('@currentRuleColumnHeading').should('be.visible');`, etc, should be enough for doing a sanity check on the headers being present.

Also, in extreme cases, that condition might fail, and I wouldn't trust Cypress to consistently maintain the correct layout:
![image](https://github.com/elastic/kibana/assets/5354282/890213cb-5270-4a9f-9c0b-02221b837dd3)

Reply:
My idea was to test that "current rule" content is shown in the "current rule" column, and same for "Elastic update". But perhaps you're right, it just overcomplicates things without adding much benefit. I'll remove this part.